tools/JsonCli/.azure-pipelines/main.yml

```diff
@@ -17,14 +17,9 @@ jobs:
       TeamName: 'AzureTools'
 
   - task: UseDotNet@2
-    displayName: 'Use .NET Core sdk 2.0.x'
+    displayName: 'Use .NET Core sdk 3.1.x'
     inputs:
-      version: 2.0.x
-
-  - task: UseDotNet@2
-    displayName: 'Use .NET Core sdk 3.0.x'
-    inputs:
-      version: 3.0.x
+      version: 3.1.x
 
   - task: UseDotNet@2
```

Comment:
Not entirely related, but you should be able to remove `includePreviewVersions: true` from the .NET 6 task since its no longer preview